### ISSUE TITLE
rename Double type, "Float", to a more informative type 

### ISSUE THREAD

**Issue by cartazio** [idris-lang/Idris-dev#379]
Either "Double", or as some folks suggested on IRC, Float64. (latter is More systematic, so i'd be ok with ). Point being Float is needlessly misleading and does't communicate correctly that nature of the float model

Should I email the idris list for this? I'm happy to write the patch for this once its sorted out.

**Comment by cartazio** [comment]
Wait, did you add 32 bit floats and rename the float type to double?

**Comment by jfdm** [comment]
@cartazio In Commit 89e325c @edwinb renamed the name of `Float` to `Double` so that it is more informative, thus addressing this issue. I  have added a line into the FAQ describing the reasoning behind the naming convention used and why a `Float64` name wasn't provided. I also recommended that this issue has been closed. Moreno, it is to the best of my knowledge that a `Float` primitive has not been added, and that the type alias called `Float` is there for backwards compatibility until the next release of Idris. 

Whether we need a size 32 float type is not my call.
